### PR TITLE
[docker] Split dependencies from tuttleofx build

### PR DESCRIPTION
**Pull request by fabiencastan**

### CODE REVIEW

File: Dockerfile-python3 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM tuttleofx/tuttleofx-env:python3-latest
```
Comment (aoblet) [inline Dockerfile-python3:1]: Including the python minor version is maybe clearer because it depends on the OS distrib image. Don't you think ?

**Comment by cchampet** [comment]
Error when launch c++ tests:

```
19: Test command: /home/travis/build/tuttleofx/TuttleOFX/testBin/plugin_io_raw
19: Test timeout computed to be: 9.99988e+06
19: Error: Missing feature: "Parametric parameter"
19: boost::exception error on action: OfxActionDescribe
19: [2016-05-30 11:01:29.033536] [0x000002b4219e6898] [trace]   [OpenFX] Loading plugins
19: [2016-05-30 11:01:29.033600] [0x000002b4219e6898] [trace]   TUTTLE_PLUGIN_PATH
19: Running 2 test cases...
19: ����������������unknown location(0): fatal error in "process_reader": memory access violation at address: 0x200000000: no mapping at fault address
```

File: Dockerfile-python3 (hunk 1 above)
Comment (fabiencastan) [inline Dockerfile-python3:1]: We only provide multiple images for the major versions, so it's easier to maintain like that.

File: tools/travis/install_dependencies.sh [hunk 2]
```diff
@@ -5,87 +5,87 @@ set -e
...
-if [[ ${TRAVIS_OS_NAME} == "linux" ]]; then
-
-    # Install python packages to run sam command line
-    if [[ ${PYTHON_VERSION} == "2.7" ]]; then
-        pip install --user clint argcomplete
-    # elif [[ ${PYTHON_VERSION} == "3.2" ]]; then
-        # easy_install3 pip --prefix=$HOME/local
-        # pip3 install --user clint argcomplete
+# Use CONTINUOUS_INTEGRATION environment variable to detect if the script is run on Travis CI.
+if [ -z ${CONTINUOUS_INTEGRATION} ]; then
```
Comment (cchampet) [inline tools/travis/install_dependencies.sh]: Should be:

```
if [ -n "${CONTINUOUS_INTEGRATION}" ]; then
```

Comment (fabiencastan) [reply tools/travis/install_dependencies.sh]: fixed

**Pull request merged**